test/cli/power_user_cmd_test.go

```diff
@@ -21,13 +21,13 @@ func TestPowerUserCmdFlags(t *testing.T) {
 				assertFailingReturnCode,
 			},
 		},
-		{
-			name: "json-output-flag-fails",
-			args: []string{"power-user", "-o", "json", "docker-archive:" + getFixtureImage(t, "image-pkg-coverage")},
-			assertions: []traitAssertion{
-				assertFailingReturnCode,
-			},
-		},
+		//{
```

Comment:
Because poweruser is being decorated with the packages flags output is now a valid option. 

TODO: update the `writer` function so poweruser can do multiple formats. This test becomes invalid.

Reply:
I want to make sure I follow what's happening here... I know we plan to remove the `power-user` command in the future. And IMHO it's not so bad if we support `-o json` for the command in the meantime (curious for your thoughts on this!).

Should we remove this test case if we don't plan to use it either now or later?

Reply:
Good callout here as I did not circle back on this comment. The solution here is to just make -o json the ONLY valid format for power-user. We can then remove this test.

It's currently hard coded here:
https://github.com/anchore/syft/blob/c270ee2a023f2ed3bacbd847bbeb6583f7db9ddc/cmd/power_user.go#L76-L81